importlib_metadata/_compat.py

```diff
@@ -52,5 +51,7 @@ def pypy_partial(val):
 
     Workaround for #327.
     """
+    import platform
```

Comment:
This seems to fail the code coverage check (diffcov)

https://github.com/python/importlib_metadata/actions/runs/10581007642/job/29317281062#step:5:102

Reply:
That's okay. It's due to there not being coverage aggregation (https://github.com/jaraco/skeleton/issues/130). It's ignorable and will disappear after merging.